include/morphio/errorMessages.h

```diff
@@ -13,6 +13,7 @@ namespace morphio {
    Set the maximum number of warnings to be printed on screen
 **/
 void set_maximum_warnings(int n_warnings);
+void raise_warnings(bool is_raise);
```

Comment:
for homogeneity I think you should name it : 
`set_raise_warnings`

Reply:
indeed, thank you!